roles/confluent.ksql/templates/ksql-server_log4j.properties.j2

```diff
@@ -45,12 +45,67 @@ log4j.appender.kafkaAppender.MaxFileSize={{ksql.kafka_appender_log_file_size}}
 log4j.appender.kafkaAppender.MaxBackupIndex={{ksql.kafka_appender_max_log_files}}
 log4j.appender.kafkaAppender.append=true
 
+{% if ksql_log_streaming_enabled | bool %}
+# Log Streaming Configuration
 log4j.appender.kafka_appender=org.apache.kafka.log4jappender.KafkaLog4jAppender
 log4j.appender.kafka_appender.layout=io.confluent.common.logging.log4j.StructuredJsonLayout
-log4j.appender.kafka_appender.BrokerList=localhost:9092
-log4j.appender.kafka_appender.Topic=default_ksql_processing_log
+log4j.appender.kafka_appender.BrokerList={% for host in groups['kafka_broker'] %}{% if loop.index > 1%},{% endif %}{{ host }}:{{kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['port']}}{% endfor %}
+log4j.appender.kafka_appender.Topic={{ksql_processing_log}}
 log4j.logger.processing=ERROR, kafka_appender
 
+# Security Configuration for Log Streaming
+{% set config_prefix = '' %}
```

Comment:
it appears the config_prefix var does not need to exist

Reply:
Good catch!  It's been removed.